UnitsNet/CustomCode/Wrappers/Pressure.Wrapper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnitsNet.Units;
+using UnitsNet.CustomCode.Units;
+
+namespace UnitsNet.CustomCode.Wrappers
+{
+    /// <summary>
+    /// From gratestas in #422
+    /// Pressure is a state function, for its measurement depends on environmental factors such as ambient pressure, elevation above sea level and local weather conditions. For two persons located in the different environment to speak of the same pressure, one must relate it to a reference. There are three basis reference: absolute, gauge, vacuum
```

Comment:
Consider changing to `located in different environments to speak of the same pressure`.

Reply:
Add xmldoc list formatting here:
```xml
<list type="bullet">
<item><description>Absolute is...</description></item>
<item>...
</list>
```